sabnzbd/deobfuscate_filenames.py

```diff
@@ -97,38 +97,49 @@ def is_probably_obfuscated(myinputfilename):
         return True
 
     # these are signals for the obfuscation versus non-obfuscation
-    capitals = sum(1 for c in filebasename if c.isupper())
-    smallletters = sum(1 for c in filebasename if c.lower())
-    spacesdots = sum(1 for c in filebasename if c == " " or c == ".")
     decimals = sum(1 for c in filebasename if c.isnumeric())
+    upperchars = sum(1 for c in filebasename if c.isupper())
+    lowerchars = sum(1 for c in filebasename if c.islower())
+    spacesdots = sum(1 for c in filebasename if c == " " or c == ".")
 
-    if capitals >= 2 and smallletters >= 2 and spacesdots >= 1:
-        logging.debug("Not obfuscated: capitals >= 2 and smallletters >= 2  and spacesdots >= 1")
+    # Example: "Great Distro"
+    if upperchars >= 2 and lowerchars >= 2 and spacesdots >= 1:
+        logging.debug("Not obfuscated: upperchars >= 2 and lowerchars >= 2  and spacesdots >= 1")
         # useful signs in filebasename, so not obfuscated
         return False
 
-    if spacesdots > 3:
+    # Example: "this is a download"
+    if spacesdots >= 3:
         # useful signs in filebasename, so not obfuscated
-        logging.debug("Not obfuscated: spacesdots > 3")
+        logging.debug("Not obfuscated: spacesdots >= 3")
         return False
 
-    if decimals > 3 and spacesdots > 1:
+    # Example: "Beast 2020"
```

Comment:
This example doesn't meet the `spacedots > 1` requirement of the check (and is only considered not obfuscated as a result of the later 'Catullus' check).

Reply:
Nice find! Thanks. I'll take care of it.